AWS/legos/aws_attach_new_policy_to_user/aws_attach_new_policy_to_user.py

```diff
@@ -0,0 +1,51 @@
+##  Copyright (c) 2021 unSkript, Inc
+##  All rights reserved.
+##
+from typing import List, Dict
+from pydantic import BaseModel, Field
+from botocore.exceptions import ClientError
+import pprint
+from beartype import beartype
+
+class InputSchema(BaseModel):
+    UserName: str = Field(
+        title='User Name',
+        description='IAM user whose policies need to fetched.')
+    PolicyName: str = Field(
+        title='Policy Name',
+        description='Policy name to apply the permissions to the user.')
+
+@beartype
+def aws_attache_iam_policy_printer(output):
+    if output is None:
+        return
+    pprint.pprint(output)
+
+
+@beartype
+def aws_attache_iam_policy(handle, UserName: str, PolicyName: str) -> Dict:
+    """aws_attache_iam_policy used to provide user permissions.
+
+        :type handle: object
+        :param handle: Object returned from task.validate(...).
+
+        :type UserName: dict

```

Comment:
UserName is supposed to be String. You are mentioning Dict. Please fix the comments.